Scenarios/TripPin/src/odatalib/.vs/config/applicationhost.config

```diff
@@ -163,7 +163,7 @@
             </site>
             <site name="Microsoft.Test.OData.Services.ODataWCFService" id="2">
                 <application path="/" applicationPool="Clr4IntegratedAppPool">
-                    <virtualDirectory path="/" physicalPath="D:\git\t\ODataSamples\Scenarios\TripPin\src\odatalib\ODataWCFService" />
+                    <virtualDirectory path="/" physicalPath="E:\OData\GitHub\OData\ODataSamples\Scenarios\TripPin\src\odatalib\ODataWCFService" />
```

Comment:
It is and I don't think this file should even be in the repo. For now, I've reverted my change to this file.